matrix_client/api.py

```diff
@@ -241,6 +241,29 @@ def send_message_event(self, room_id, event_type, content, txn_id=None,
             params["ts"] = timestamp
         return self._send("PUT", path, content, query_params=params)
 
+
+    def send_redact_event(self, room_id, event_id, content, txn_id=None, timestamp=None):
```

Comment:
Can we make this `redact_event` please, for the sake of terseness? I see why
you did it to match the other nearby `send_*_event`, but I think
`redact_event` would be clearer and simpler here.